globals/plotData.json

```diff
@@ -0,0 +1,256 @@
+{
+  "central_occupation": {
+    "label": "Central Tracer Occupation",
+    "scale": "log",
+    "name": "Occupation of central component"
+  },
+  "cmz_relation": {
+    "label": "Halo Concentration",
+    "scale": "log",
+    "name": "Halo Concentration-Mass Relation"
+  },
+  "corr_1h_auto_matter": {
+    "label": "1-halo matter correlation, $\\xi_{1h}(r)$",
+    "scale": "log",
+    "name": "1-halo matter-matter correlation function"
+  },
+  "corr_1h_auto_tracer": {
+    "label": "1-halo tracer correlation, $\\xi_{1h}(r)$",
+    "scale": "log",
+    "name": "1-halo tracer-tracer correlation function"
+  },
+  "corr_1h_cross_tracer_matter": {
+    "label": "1-halo matter-tracer correlation, $\\xi_{1h}^{m\\times T}(r)$",
+    "scale": "linear",
+    "name": "1-halo matter-tracer correlation function"
+  },
```

Comment:
How will the tests access these? As well as the deployed front-end and backend. This might work locally, but when it is running in CI, normally it is isolated from anything higher than the folder that is used to deploy the server/front-end.

Reply:
For webpack stuff, it's actually ok, it'll be "compiled in"
Idk how python stuff will do though. The docker compose or docker files will need to copy it in.

Reply:
The pytests and all works right now. I'm not sure about deployment though..

Reply:
@dblevin1

Reply:
Okay it is looking like it is coming along then. Just a few more comments below 😄